src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -101,6 +102,18 @@
     <Reference Include="NuGet.ProjectModel">
       <HintPath>..\..\packages\NuGet.ProjectModel.4.6.0\lib\net46\NuGet.ProjectModel.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Build.Framework">
+      <HintPath>..\..\packages\Microsoft.Build.Framework.14.3.0\lib\net45\Microsoft.Build.Framework.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Build.Utilities.Core">
+      <HintPath>..\..\packages\Microsoft.Build.Utilities.Core.14.3.0\lib\net45\Microsoft.Build.Utilities.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Build.Tasks.Core">
+      <HintPath>..\..\packages\Microsoft.Build.Tasks.Core.14.3.0\lib\net45\Microsoft.Build.Tasks.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="Xamarin.Build.AsyncTask">
+      <HintPath>..\..\packages\Xamarin.Build.AsyncTask.0.3.4\lib\netstandard2.0\Xamarin.Build.AsyncTask.dll</HintPath>
+    </Reference>
```

Comment:
Ok, so these mean we have two references to these libraries we import these:

https://github.com/xamarin/xamarin-android/blob/879afff56be70930448a9ba6892eb0e7ac13db83/build-tools/scripts/MSBuildReferences.projitems#L8-L22

Do we need to bring in the ones from NuGet at all?

Reply:
It's a Dependency of the `AsyncTask` package. not sure can even stop it bringing them in?